tests/wpt/web-platform-tests/html/syntax/serializing-html-fragments/serializing.html

```diff
@@ -59,7 +59,7 @@
 ["\"", "<span>\"</span>"],
 ["<style><&></style>", "<span><style><&></style></span>"],
 ["<script type=\"test\"><&><\/script>", "<span><script type=\"test\"><&><\/script></span>"],
-["<script type=\"test\"><&><\/script>", "<script type=\"test\"><&><\/script>"],
+["<script><&><\/script>", "<&>"],
```

Comment:
Aren't these two values reversed? The first value is the expected output of innerHTML, the second is the expected output of outerHTML.